app/assets/javascripts/manage_submissions.js

```diff
@@ -1,3 +1,17 @@
+var submission_info = {}
+let tweaks = [];
+const Edit = (totalSum) => {
+  if (totalSum == null) {
+    return `
+      <span>-</span>
+      <i class="material-icons" style="color:#6F6F6F; margin-left:3px">edit</i>
```

Comment:
nit: can we use scss variables here (and below)? Like $autolab-blue-text and maybe $autolab-subtle-gray or $autolab-highlight-gray